### PR TITLE
Welcome screen

### PR DESCRIPTION
**Pull request by MRandl**

### CODE REVIEW

**Review by AttiaYoussef — APPROVED** [review]

File: app/src/main/res/layout/activity_no_connection.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.19"
+        app:srcCompat="@android:drawable/presence_busy" />
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
```
Comment (MrZaiko) [inline app/src/main/res/layout/activity_no_connection.xml]: Not a very good id

File: app/src/main/res/layout/activity_no_connection.xml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".login.NoConnectionActivity">
+
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="200dp"
```
Comment (MrZaiko) [inline app/src/main/res/layout/activity_no_connection.xml]: same

File: app/src/main/res/layout/activity_no_connection.xml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/loginNoConnection"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:id="@+id/button2"
```
Comment (MrZaiko) [inline app/src/main/res/layout/activity_no_connection.xml]: Same

**Review by MrZaiko — CHANGES_REQUESTED** [review]

**Review by AttiaYoussef — CHANGES_REQUESTED** [review]
Bad ids

**Review by AttiaYoussef — APPROVED** [review]

**Review by MrZaiko — APPROVED** [review]
Good work

**Pull request merged**